handlers/security.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module Lita
+  module Handlers
+    # some comment
+    class Security < Handler
+      config :github, default: Zooniverse::Github.new
+
+      route(/^(dependabot)\s*(.*)/, :get_dependabot_issues, command: true,
+                                                            help: { 'status dependabot' => 'displays dependabot security alerts' })
+
+      def get_dependabot_issues(response)
+        get_issues = true
+        last_repo_listed = nil
+        alerts = []
+        repo_to_alert_count = {}
+
+        while get_issues == true
+          res = config.github.get_dependabot_issues(last_repo_listed)
+          if res.nil?
+            get_issues = false
+            break
+          end
+          edges = res['data']['organization']['repositories']['edges']
+          nodes = res['data']['organization']['repositories']['nodes']
+          nodes.each do |node|
+            node_alerts = node['vulnerabilityAlerts']['nodes']
+            repo_name = node['name']
+            next if repos_to_skip.include? repo_name
+            next if node_alerts.empty?
+
+            filter_alerts(node_alerts, alerts, repo_to_alert_count, repo_name)
+          end
+          repo_count = edges.length
+          last_repo_listed = edges[repo_count - 1]['cursor']
+          get_issues = false if repo_count < 100
+        end
+
+        response.reply("#{alerts.length} Alerts Total: \n #{format_alerts(repo_to_alert_count)}")
+      end
+
+      private
+
+      def filter_alerts(node_alerts, alerts, repo_to_alert_count, repo_name)
+        node_alerts.each do |alert|
+          next unless alert['dismissedAt'].nil?
+          next unless alert['fixedAt'].nil?
```

Comment:
no need for the nil tests here as it makes the conditions hard to read / understand if not value is nil? 
```suggestion
          next if alert['dismissedAt']
          next if alert['fixedAt']
```